configs/crashtests-webrtc.ini

```diff
@@ -0,0 +1,28 @@
+[runtests]
+# Settings related to executing runtestsremote.py
+
+# test_name is a descriptor used by runtestsremote.py to
+# determine which of the downloaded unit tests to run.
+#
+# test_manifest is the path to the location of the test_manifest.
+#
+# test_package_names is a list of the keys for test_packages.json
+# which will need to be downloaded in order to run the test.
+
+test_name = crashtest
```

Comment:
This needs to be distinct from the other tests. crashtest-webrtc should do it. The full formal name will end up being autophone-crashhtest-webrtc when we want to put this in trychooser.